external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaMicroBatchSourceSuite.scala

```diff
@@ -1511,6 +1515,54 @@ abstract class KafkaSourceSuiteBase extends KafkaSourceTest {
       CheckAnswer(2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17)
     )
   }
+
+  protected def testSpark27494(trigger: Trigger): Unit = {
```

Comment:
Can we have a meaningful name instead of `testSpark27494`?

Reply:
what about `testNullableKeyValue`?

Reply:
Yep. That looks better to me.